src/main/java/frc/robot/commands/indexer/IndexerCommand.java

```diff
@@ -12,11 +14,11 @@ public class IndexerCommand extends Command {
 	private final IndexerSubsystem m_indexerSubsystem;
 
 	public static IndexerCommand getFowardCommand(IndexerSubsystem indexerSubsystem) {
-		return new IndexerCommand(indexerSubsystem, -.8);
```

Comment:
Forward? Reverse? Which is it?

Reply:
That was because the motor should have been inverted, if you look at it I put that number to positive and inverted the motor

Reply:
Oh wait I see what you mean lmfao

Reply:
I'll fix that

Reply:
Done